components/IssueCredentialOptions.yml

```diff
@@ -1,40 +1,43 @@
-openapi: 3.0.0
-info:
-  version: "0.0.3-unstable"
-  title: VC API
-  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
-  license:
-    name: W3C Software and Document License
-    url: http://www.w3.org/Consortium/Legal/copyright-software.
-  contact:
-    name: GitHub Source Code
-    url: https://github.com/w3c-ccg/vc-api
-paths:
-components:
-  schemas:
-    IssueCredentialOptions:
-      type: object
-      additionalProperties: false
-      description: Options for specifying how the DataIntegrityProof is created.
-      properties:
-        created:
-          type: string
-          description: The date and time of the proof (with a maximum accuracy in seconds). Default current system time.
-        challenge:
-          type: string
-          description: A challenge provided by the requesting party of the proof. For example 6e62f66e-67de-11eb-b490-ef3eeefa55f2
-        domain:
-          type: string
-          description: The intended domain of validity for the proof. For example website.example
-        credentialStatus:
-          type: object
-          description: The method of credential status to issue the credential including. If omitted credential status will be included.
-          properties:
-            type:
-              type: string
-              description: The type of credential status to issue the credential with
-      example:
-        {
-          "created": "2020-04-02T18:48:36Z",
-          "credentialStatus": { "type": "RevocationList2020Status" },
-        }
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    IssueCredentialOptions:
+      type: object
+      additionalProperties: false
+      description: Options for specifying how the DataIntegrityProof is created.
+      properties:
+        created:
+          type: string
+          description: The date and time of the proof (with a maximum accuracy in seconds). Default current system time.
+        challenge:
+          type: string
+          description: A challenge provided by the requesting party of the proof. For example 6e62f66e-67de-11eb-b490-ef3eeefa55f2
+        domain:
+          type: string
+          description: The intended domain of validity for the proof. For example website.example

```

Comment:
These are not options for issuance (they are for authentication proofs that holders generate), we could remove them in this PR or another later.

Reply:
I think these are re-used in presentation creation, which is why they're here. Agree that we probably want to differentiate the Credential issuance from presentation creation/issuance functions.